fastlane/Jetpack-Fastfile

```diff
@@ -17,12 +17,21 @@ APP_IDENTIFIER = 'com.automattic.jetpack'
   desc "Builds and uploads a Jetpack installable build"
   lane :build_and_upload_jetpack_installable_build do | options |
     jetpack_alpha_code_signing
+    require 'git'
 
     # Get the current build version, and update it if needed
     version_config_path = File.join(PROJECT_ROOT_FOLDER, "config", "Version.internal.xcconfig")
     versions = Xcodeproj::Config.new(File.new(version_config_path)).to_hash
 
-    build_number = Time.new.strftime("%Y-%m-%d-%H")
+    long_hash = Git.open(PROJECT_ROOT_FOLDER).revparse('HEAD')
+    short_hash = `git rev-parse --short #{long_hash}`.strip!
```

Comment:
We have a helper method in release-toolkit which does something similar. Worth exposing as an action so it can be access via our Fastfiles? 🤔

Reply:
I thought about this, but [the current implementation](https://github.com/wordpress-mobile/release-toolkit/blob/68f3356b1578935e8368f59ac6c32bbe14327bb3/lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb#L129-L136) returns a full hash, and the `ruby-git` project doesn't expose a nice interface for this.

Doesn't mean we couldn't fix it later, but for now I'm fine with this as a one-off? 🤷

Reply:
> Doesn't mean we couldn't fix it later, but for now I'm fine with this as a one-off? 🤷

👍 worth adding an issue / note somewhere? It's the kind of thing that would be nice to have, but easy enough to roll out manually. Meaning that the next time someone needs this, unless they know about / remember this conversation, they're likely to add yet another manual call.

Reply:
I mean, better than that, we could have used the built-in action already existing in fastlane for that apparently…

```
$ fastlane actions | grep git
…
| last_git_commit                             | Return last git commit hash, abbreviated commit hash, commit      | ngutman          |
…

$ fastlane action last_git_commit
…

+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                                              last_git_commit Return Value                                                                              |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Returns the following dict: {commit_hash: "commit hash", abbreviated_commit_hash: "abbreviated commit hash" author: "Author", author_email: "author email", message: "commit message"} |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Reply:
```suggestion
    short_hash = last_git_commit[:abbreviated_commit_hash]
```